samplemanager/sample_manager.py

```diff
@@ -41,7 +41,7 @@ def __init__(self):
             "Create a production file",     # Task 5
             "Update genweight",             # Task 6
             "Update xsec",                  # Task 7
-            "Maintainance",                 # Task 8
+            "Maintenance",                 # Task 8
```

Comment:
Just because I like symmetries, add an empty space here to align the task comments.